src/app/notifications/notifications.component.html

```diff
@@ -0,0 +1,16 @@
+<div *ngFor="let notification of allNotifications$ | async">
```

Comment:
should be `notDismissedNotifications` instead of `allNotifications` here because your `*ngIf` down there is currently being executed a bajillion times